example/main.tf

```
@@ -0,0 +1,15 @@
+# Provision group access to data account
+module "organization_access_group_data" {
+  source      = "../"
+  enabled     = "true"
+  namespace   = "cp"
```

```suggestion
  namespace   = "eg"
```

Let's not use `cp` for the namespace.
People copy the examples verbatim and get errors that some global AWS resources with that name already exist